src/v1/poke_battle.py

```diff
@@ -63,14 +74,29 @@ def calc_battles():
         poke_one = pokemons[pokemons["Id"] == first_pokemon]
         poke_two = pokemons[pokemons["Id"] == second_pokemon]
 
-        pokemon_battle_info = pd.Series([winner,poke_one['HP'].values[0],poke_one['Attack'].values[0],poke_one['Defense'].values[0],poke_one['Sp_Attack'].values[0],poke_one['Sp_Defense'].values[0],poke_one['Speed'].values[0],poke_two['HP'].values[0],poke_two['Attack'].values[0],poke_two['Defense'].values[0],poke_two['Sp_Attack'].values[0],poke_two['Sp_Defense'].values[0],poke_two['Speed'].values[0]])
-        battle_to_concat = pd.DataFrame([pokemon_battle_info])
+        win = did_1_win(winner, first_pokemon)
+
+        hp_diff = calc_stat_diff(poke_one['HP'].values[0],poke_two['HP'].values[0])
+        attack_diff = calc_stat_diff(poke_one['Attack'].values[0],poke_two['Attack'].values[0])
+        defense_diff = calc_stat_diff(poke_one['Defense'].values[0],poke_two['Defense'].values[0])
+        sp_attack_diff = calc_stat_diff(poke_one['Sp_Attack'].values[0],poke_two['Sp_Attack'].values[0])
+        sp_defense_diff = calc_stat_diff(poke_one['Sp_Defense'].values[0],poke_two['Sp_Defense'].values[0])
+        speed_diff = calc_stat_diff(poke_one['Speed'].values[0],poke_two['Speed'].values[0])
+
+        stats = pd.Series([win, hp_diff, attack_diff, defense_diff, sp_attack_diff, sp_defense_diff, speed_diff])
```

Comment:
@Freddiiy @bentsen jeg har lavet om på hvordan dataen bliver oprettet. Istedet for at vores winner er et pokemon id (1-151), så returnerer den 1 hvis det er pokemon 1 der vandt, eller 0 hvis det var pokemon 2.
Udover det så er det differencen mellem der stats der bliver lagt der, så hvis poke1 havde højere HP stat ville den være positiv (f.eks 20) hvis poke2 havde højere HP stat ville den være negativ (f.eks -20).